models.py

```diff
@@ -48,4 +49,30 @@ def forward(self, x, pad_mask):
         x = self.output_norm(x)
         x = self.output(x)
         return torch.squeeze(x)
-    
\ No newline at end of file
+
+class WatsonCrickAttentionLayer(torch.nn.Module):
+
+    def __init__(self, size, score_matrix):
```

Comment:
not sure why score_matrix needs to be passed in? this should probably just be randomly initialized like any other params